<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>DoctoreditView</class>
<widget class="QWidget" name="DoctoreditView">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>500</width>
   <height>550</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>医生信息编辑</string>
 </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <widget class="QGroupBox" name="groupBox">
    <property name="title">
     <string>基本信息</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <!-- 医生ID显示（第一行） -->
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>医生ID：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtDoctorId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>自动生成</string>
       </property>
      </widget>
     </item>
     
     <!-- 工号（第二行） -->
     <item row="1" column="0">
      <widget class="QLabel" name="label_employee_id">
       <property name="text">
        <string>工号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtEmployeeId"/>
     </item>
     
     <!-- 姓名（第三行） -->
     <item row="2" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     
     <!-- 性别（第四行） -->
     <item row="3" column="0">
      <widget class="QLabel" name="label_gender">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmbGender"/>
     </item>
     
     <!-- 出生日期（第五行） -->
     <item row="4" column="0">
      <widget class="QLabel" name="label_birth">
       <property name="text">
        <string>出生日期：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateBirth"/>
     </item>
     
     <!-- 科室（第六行） -->
     <item row="5" column="0">
      <widget class="QLabel" name="label_department">
       <property name="text">
        <string>科室：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cmbDepartment"/>
     </item>
     
     <!-- 职称（第七行） -->
     <item row="6" column="0">
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>职称：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cmbTitle"/>
     </item>
     
     <!-- 专业方向（第八行） -->
     <item row="7" column="0">
      <widget class="QLabel" name="label_specialization">
       <property name="text">
        <string>专业方向：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="txtSpecialization"/>
     </item>
     
     <!-- 联系电话（第九行） -->
     <item row="8" column="0">
      <widget class="QLabel" name="label_phone">
       <property name="text">
        <string>联系电话：</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="txtPhone"/>
     </item>
     
     <!-- 邮箱（第十行） -->
     <item row="9" column="0">
      <widget class="QLabel" name="label_email">
       <property name="text">
        <string>邮箱：</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="txtEmail"/>
     </item>
     
     <!-- 工作年限（第十一行） -->
     <item row="10" column="0">
      <widget class="QLabel" name="label_work_years">
       <property name="text">
        <string>工作年限：</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="spinWorkYears">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     
     <!-- 诊费（第十二行） -->
     <item row="11" column="0">
      <widget class="QLabel" name="label_fee">
       <property name="text">
        <string>诊费：</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="spinFee">
       <property name="minimum">
        <double>0.00</double>
       </property>
       <property name="maximum">
        <double>1000.00</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="suffix">
        <string> 元</string>
       </property>
      </widget>
     </item>
     
     <!-- 状态（第十三行） -->
     <item row="12" column="0">
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="cmbStatus"/>
     </item>
    </layout>
   </widget>
  </item>
  
  <!-- 错误信息 -->
  <item>
   <widget class="QLabel" name="lblError">
    <property name="text">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </item>
  
  <!-- 按钮区域 -->
  <item>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>